.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
```

Comment:
I assume this is autogenerated from somewhere? This repo was migrated to Github Actions as part of https://github.com/gocardless/gocardless-pro-php/pull/145

Reply:
cc @sam-robson

Reply:
good spot I reckon its because the commit that removed the circle ci config failed here https://github.com/gocardless/gocardless-pro-php-template/commit/b12aa43e074500c2b8f26b1ba669e7947fb44ffc

so the template commit to this branch didnt happen properly Ill re run the CI on the template branch it should re sync 👌

Reply:
found it,not all circle ci config was fully cleaned up I've removed it here https://github.com/gocardless/gocardless-pro-php-template/pull/196 changes should sync through shortly

Reply:
nvm its bugged and the latest commit that got triggered just has some old changes instead of my new template ones 😓

Reply:
I've removed it on both sides so we should be good now